public/styles/app.css

```diff
@@ -0,0 +1,3 @@
+.color{
+    background: beige;
```

Comment:
I don't see where this is used.

Reply:
New packages must be added, which I will do that later.

Reply:
Marking as resolved then.